index.js

```diff
@@ -12,3 +14,15 @@ const pagination = document.querySelector('[data-js="pagination"]');
 const maxPage = 1;
 const page = 1;
 const searchQuery = "";
+
+async function fetchCharacters() {
+  try {
+    const response = await fetch("https://rickandmortyapi.com/api/character/");
+    const data = await response.json();
+    console.log(data.results);
+    return data.results;
```

Comment:
The fetch looks good. However, instead of just returning the data, you should also create the character cards with the data. For simplicity, I recommend doing this directly within the fetch function. Since you have divided the tasks among your group, I suggest the following steps to complete this task:

	1.	Merge the CharacterCard branch into main.
	2.	Then merge the current state of the main branch into the APIFetchData branch. This way, you can use the CharacterCard component in this branch to create cards for the fetched data.
	3.	Please remember to clear the CardContainer as described in the task instructions.